ProprioceptiveState proprio  
AchievedGoal ag
ResetInfo resetState
sensor_msgs/Image shoulderImage
sensor_msgs/Image gripperImage
PositionCommand a
int32 timestep
float64 beat_sent_time
float64 data_arrival_time
float64 model_processed_time

